docs/usage/configuration-options.md

```diff
@@ -1412,14 +1412,21 @@ This feature can be used to refresh lock files and keep them up-to-date.
 "Maintaining" a lock file means recreating it so that every dependency version within it is updated to the latest.
 Supported lock files are:
 
-- `package-lock.json`
-- `yarn.lock`
-- `pnpm-lock.yaml`
+- `.terraform.lock.hcl`
+- `Cargo.lock`
 - `composer.lock`
+- `flake.lock`
 - `Gemfile.lock`
-- `poetry.lock`
-- `Cargo.lock`
+- `gradle.lockfile`
 - `jsonnetfile.lock.json`
+- `package-lock.json`
+- `Pipfile.lock`
+- `pnpm-lock.yaml`
+- `poetry.lock`
+- `pyproject.toml`
+- `requirements.lock` // Helm V3 lockfile?
```

Comment:
```suggestion
- `Chart.lock`
- `Chart.yaml`
```

Are these the right lockfiles for Helm v3?

Do we support Helm V2 (the older version)? If yes, maybe we should keep the `requirements.yaml` and `requirements.lock` in the list of supported lockfiles? 

Quote from a Helm v3 preview blog: [^blog]

> The Chart dependency management system moved from `requirements.yaml` and `requirements.lock` to `Chart.yaml` and `Chart.lock`, meaning that charts that relied on the helm dependency command will need some tweaking to work in Helm 3.

[^blog]: https://helm.sh/blog/helm-3-preview-pt5/#helm

Reply:
Sounds like just `Chart.lock`